### ISSUE TITLE
Give bigSNP object genotype col/row names?

### ISSUE THREAD

**Issue by Jocelyn37** [privefl/bigsnpr#397]
Heyy, I wanna ask is there a way to give bigSNP genotypes data "FBM.code256" class the rownames and colnames? Now I found out I could only give it row/colnames after I converted it into matrix, but the matrix takes up so many memory, so I am trying to find out the way to give it row/colnames directly to bigSNP genotypes data. Thank you!

**Comment by privefl** [comment]
No, it is not possible, and I probably won't implement this anytime soon (cf. https://github.com/privefl/bigstatsr/issues/64).
Sorry

The simplest workaround would be to store row IDs in `$fam` and col IDs in `$map` and use `match()` to get the indices corresponding to the names you want to use.

**Comment by Jocelyn37** [comment]
Okay thank you very much!!!
